channels/sl-micro-60-baremetal.json

```diff
@@ -8,7 +8,8 @@
             "type": "container",
             "metadata": {
                 "upgradeImage": "registry.suse.com/suse/sl-micro/6.0/baremetal-os-container:2.1.1-3.29",
-                "displayName": "SL Micro 6.0 OS"
+                "displayName": "SL Micro 6.0 OS",
+                "architectures": ["amd64","arm64"]
```

Comment:
Are we sure we want to use this notation? I wonder if it should be `x86_64` and `aarch64` 🤔 Is this the notation form the manifest? If so I'd probably stick to that.

I could not find any architecture reference in elemental-operator so from that PoV we have don't have any precedent on which notation to use. In toolkit I think we have slightly preference for `x86_64` and `aarch64`, however both notations are in use in toolkit internally as different tools and libraries require one or the other.

Reply:
Indeed this is what comes out the manifest:

```
$ skopeo inspect --raw docker://registry.suse.com/suse/sl-micro/6.0/baremetal-os-container:2.1.1-3.29
{
   "schemaVersion" : 2,
   "mediaType" : "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests" : [
      {
         "mediaType" : "application/vnd.docker.distribution.manifest.v2+json",
         "size" : 757,
         "digest" : "sha256:158fc96ffca227c330b66d2dbe0483dc28b9ed80765b3c9045362da569eff0e6",
         "platform" : {
            "architecture" : "amd64",
            "os" : "linux"
         }
      },
      {
         "mediaType" : "application/vnd.docker.distribution.manifest.v2+json",
         "size" : 757,
         "digest" : "sha256:f0f03a9fa7bbee92b6ccf34a67dc9b70c704df1456abd550abe5a47bc3de0cf2",
         "platform" : {
            "architecture" : "arm64",
            "os" : "linux",
            "variant" : "v8"
         }
      }
   ]
}
```

Reply:
Still I can very easily sed it to our wanted values. 
However note that the `targetPlatform` field in elemental-operator also uses the `amd64`/`arm64` notation :D

https://elemental.docs.rancher.com/seedimage-reference

The targetPlatform also includes the `os`:`linux` which I didn't include in the managedosversion. We can add it, but due to the nature of Elemental, I doubt we will support anything else any time soon.

Reply:
Last but not least, if we want to stick to a certain notation this is the right time, since 1.6 is not out yet and `targetPlatform` can be considered a not yet released feature.